### PR TITLE
template rename changes

### PR DESCRIPTION
**Pull request by wanlonghenry**
rename changes for provisioned cluster onboarding

### CODE REVIEW

File: scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json [hunk 1]
```diff
@@ -237,7 +237,7 @@
                           "mode": "Incremental",
                           "id": "[parameters('aksResourceId')]",
                           "addonProfiles": {
```
Comment (ganga1980) [inline scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json:239]: No changes required. This has to be omsagent for AKS.

File: test/onboarding-templates-legacy-auth/existingClusterOnboarding.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
                 "mode": "Incremental",
                 "id": "[parameters('aksResourceId')]",
                 "addonProfiles": {
-                    "omsagent": {
+                    "amalogs": {
```
Comment (ganga1980) [inline test/onboarding-templates-legacy-auth/existingClusterOnboarding.json:34]: No changes to aks addon profile. Please revert this change.

File: scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json (hunk 1 above)
Comment (wanlonghenry) [reply scripts/onboarding/aks/onboarding-using-msi-auth/existingClusterOnboarding.json]: removed

**Review by ganga1980 — APPROVED** [review]
